opentracing-flowid/opentracing-flowid/src/test/java/org/zalando/opentracing/flowid/DefaultFlowNoopTracerTest.java

```diff
@@ -12,4 +17,11 @@
     void shouldThrowNoActiveSpanFoundException() {
```

Comment:
Not throwing exceptions is the right direction here - can we also make this test now throw?

Reply:
Not throw?
If you tell me what `currentId()` should return when there is no active span, sure. Just null?